server/routes/caseAdmin/assessment.ts

```diff
@@ -12,12 +10,6 @@ export default class AssessmentRoutes {
       req.params.prisonNumber,
     )
 
-    res.render('pages/caseAdmin/assessment', {
-      assessmentSummary: {
-        ...assessmentSummary,
-        name: convertToTitleCase(`${assessmentSummary.forename} ${assessmentSummary.surname}`.trim()),
```

Comment:
I've pushed name building into the template and there's a `toFullName` filter than can be applied to assessment summaries

Reply:
nice, I've been meaning to do something like that for a while as that was duplicated on just about page